modules/CONT-05_Merging-pull-requests.yml

```diff
@@ -46,7 +46,7 @@ screens:
       presenter-script:
         - Let's finish the workflow by merging our changes into master.
       steps:
-        - description: "Merge the pull request you created, closing your original issue with the merge commit."
+        - description: "Merge the pull request you created, closing your original issue with the merge commit. Don't forget Github looks for the 'Fixes' keyword."
```

Comment:
Yeah that was silly of me :smile:, it should absolutely mention all three.
Will update shortly!

On Mon, Jul 27, 2015, 9:04 AM Cynthia Rich <notifications@github.com> wrote:

> In modules/CONT-05_Merging-pull-requests.yml
> <https://github.com/certify/curriculum-github/pull/102#discussion_r35532625>
> :
>
> > @@ -46,7 +46,7 @@ screens:
> >        presenter-script:
> >          - Let's finish the workflow by merging our changes into master.
> >        steps:
> > -        - description: "Merge the pull request you created, closing your original issue with the merge commit."
> > +        - description: "Merge the pull request you created, closing your original issue with the merge commit. Don't forget Github looks for the 'Fixes' keyword."
>
> Do we specifically want to call out fixes? Or should we say something
> like, "Don't forget GitHub looks for the keywords: 'Fixes', 'Closes', or
> 'Resolves'."
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/certify/curriculum-github/pull/102/files#r35532625>.
>